benchmark/url/whatwg-url-properties.js

```diff
@@ -74,17 +62,13 @@ function main(conf) {
     case 'pathname':
     case 'search':
     case 'hash':
+    case 'href':
       setAndGet(n, url, prop, getAlternative(prop));
       break;
-    // TODO: move href to the first group when the setter lands.
-    case 'href':
     case 'origin':
     case 'searchParams':
       get(n, url, prop);
       break;
-    case 'toString':
```

Comment:
This is covered in `legacy-vs-whatwg-url-serialize.js` with `method="whatwg"`, also `href` is an alias of `toString` so we can just look at the results of `href`.